src/store/playbackManager.ts

```diff
@@ -73,6 +74,7 @@ export const defaultState = (): PlaybackManagerState => ({
   currentItemChapters: null,
   currentTime: null,
   lastProgressUpdate: 0,
+  previousValume: 100,
```

Comment:
To simpligy all of this, I would just save a boolean ``isMuted`` in the store. In the same store action you can check if the new volume === 0 and commit the mutation that toggles the ``isMuted`` variable.

In the components, just map the state of that var and bind it to the muted attribute of the video and audio element.
Thoughts @ThibaultNocchi ?

Reply:
Yup I agree